services/auth.js

```diff
@@ -90,7 +91,41 @@ const authService = {
     await tokenService.saveToken(user._id, tokens.refreshToken)
 
     return tokens
-  }
+  },
+
+  sendResetPasswordEmail: async (email, refreshToken) => {
+    const user = await User.findOne({ email }).exec()
+    if (!user) {
+      throw createError(404, EMAIL_NOT_FOUND)
+    }
+
+    if (!refreshToken) {
+      throw createUnauthorizedError()
+    }
+
+    await sendEmail(email, emailSubject.RESET_PASSWORD, { refreshToken })
```

Comment:
gotta pass were resetToken, firstName and email in {}
(check email reset template)

Reply:
Updated